multicluster/controllers/multicluster/common/helper.go

```diff
@@ -13,12 +13,9 @@ limitations under the License.
 
 package common
 
-import "k8s.io/apimachinery/pkg/types"
-
 const (
-	AntreaMcsLabel        = "antrea.io/multi-cluster"
-	AntreaMcsAutoGenLabel = "antrea.io/multi-cluster-autogenerated"
-	SourceImportLabel     = "antrea.io/resource-import-name"
+	AntreaMCSAutoGenAnnotation = "antrea.io/multi-cluster-autogenerated"
```

Comment:
What you think about "multicluster.antrea.io/imported-service" and "multicluster.antrea.io/resource-import-name"?

Reply:
ok, will do.